src/main/java/com/noroff/mefit/model/Address.java

```diff
@@ -0,0 +1,52 @@
+package com.noroff.mefit.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "address", schema = "public")
+public class Address {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter
+    @Setter
+    @Column(name = "id")
+    private Integer id;
+
+    @Getter @Setter
+    @Size(min = 1, max = 50)
+    @Column(name = "address_line_1")
+    private String addressLine1;
+
+    @Getter @Setter
+    @Size(min = 1, max = 50)
```

Comment:
remove `@Size`. Address size is generally unknown.
Or tweak values if its something like `apt. 1234` or `12` the range could be from 1 - 20 perhaps
```suggestion
@Size(min = 1, max = 20)
```
```suggestion
```